.github/workflows/client.yml

```diff
@@ -2,10 +2,10 @@ name: Client Workflow
 
 on:
   push:
-    paths:
-      - "client/**"
     branches:
       - main
```

Comment:
https://github.com/fazbazjaz/star/actions/runs/6981009111 this merged PR failed build on main.

Why don't we run the test on each branch or pr and disable merging until the test past?

Reply:
Yes good idea 😎